src/app/workspace/resource/resource-instance-form/select-properties/select-properties.component.ts

```diff
@@ -65,6 +72,23 @@ export class SelectPropertiesComponent implements OnInit {
         );
     }
 
+    isPropRequired(propId: string): void {
```

Comment:
I think it would make more sense for this method to return a boolean in case we need this logic for other booleans in the future.

Reply:
done in 7858d95